src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,488 @@
+# Microelectrode Electrophysiology
+
+Support for Microelectrode Electrophysiology was developed as a [BIDS Extension Proposal](../extensions.md#bids-extension-proposals) [BEP032: Animal electrophysiology (ephys)](https://bids.neuroimaging.io/bep032).
```

Comment:
I found "official" https://github.com/bids-standard/bids-specification/blob/HEAD/CONTRIBUTING.md#soft-rules 

> Similarly try to use "hard word wrapping": if a sentence gets long and extends a line length beyond 80-100 characters, continue the sentence on the next line.

and below there the advise seems to still keep sentence end ending the line, e.g.

> Unprocessed MEG data MUST be stored in the native file format of the MEG instrument
> with which the data was collected.
> With the MEG specification of BIDS, we wish to promote the adoption of good practices
> in the management of scientific data.

@Peyman-N could you please as a first step do such reformatting so to ease adding suggestions and minimize number of commits

Reply:
@yarikoptic I did it inn this pull request 
https://github.com/bids-standard/bids-specification/pull/2012

Reply:
Merged now, thank you!